### PR TITLE
Dp 24108 backend and frontend changes to support blog content with collections

### PR DESCRIPTION
**Pull request by beto-aveiga**
Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)

## Related Issue / Ticket

- [JIRA issue]()
- [Github issue]()

## Steps to Test

1.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application

*

#### @TODO

*

### CODE REVIEW

File: packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/blogpost-collection-header.twig [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<div class="blogpost-collection-header {{ dark_font_class }}" style="background-color: {{ blogpostCollectionHeader.bg_color }}">
+  <div class="blogpost-collection-header--inner">
+
+    <div class="blogpost-collection-header__content">
+      {% include "@atoms/decorative-link.twig" with blogpostCollectionHeader %}
+    </div>
+
+    {% if blogpostCollectionHeader.image|raw is not empty %}
+      <div class="blogpost-collection-header__image">
+        {{ blogpostCollectionHeader.image|raw }}
```
Comment (tom-fleming) [inline packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/blogpost-collection-header.twig]: @beto-aveiga Is there a reason we need to use raw here? I don't think this is something we've needed to do with other images in Mayflower.

File: packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/collection-header.twig [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      {% endif %}
+
+      {% if collection.description is not empty %}
+        <div class="collection-header__description">{{ collection.description }}</div>
+      {% endif %}
+    </div>
+
+    {% if collection.image|raw is not empty %}
+      <div class="collection-header__image">
+        {{ collection.image|raw }}
```
Comment (tom-fleming) [inline packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/collection-header.twig]: @beto-aveiga Is there a reason we need to use raw here? I don't think this is something we've needed to do with other images in Mayflower.

File: packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/blogpost-collection-header.twig (hunk 1 above)
Comment (beto-aveiga) [reply packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/blogpost-collection-header.twig]: It was a mistake. I just removed it. Thanks!

File: packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/collection-header.twig (hunk 2 above)
Comment (beto-aveiga) [reply packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/collection-header.twig]: It was a mistake. I just removed it. Thanks!

**Review by tom-fleming — APPROVED** [review]

**Pull request merged**